.env.sample

```diff
@@ -22,3 +22,5 @@ AWS_ACCESS_KEY_ID=''
 AWS_SECRET_ACCESS_KEY=''
 # Vault:
 VAULT_TOKEN=''
+
+OTOMI_CHARTS_URL=''
```

Comment:
For dev purpose let's make it to `https://github.com/redkubes/otomi-charts`.
Then workload creation will be able to go through the all steps